node-js-web-server/server090.js

```diff
@@ -0,0 +1,15 @@
+const http = require('http');
+
+http.createServer(function (request, response) {
+   console.log('request url: ', request.url);
+   
+   var filePath = './public' + request.url;
+   if (filePath == './public/') {
+      filePath = './public/index.html';
+   }
+}).listen(8080);
+console.log('Server is running at http://localhost:8080');
+
+// for this lesson require `path` module in your code
```

Comment:
We will need the `path` module to read the extensions of our files later.
Just under your `http` variable, `require` the `path` module and assign it to a variable named `path`.